apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java

```diff
@@ -575,25 +514,6 @@ private List<RifRecordLoadResult> process(
       LOGGER.warn("Failed to load '{}' record.", rifFileType, t);
 
       throw new RifLoadFailure(recordsBatch, t);
-    } finally {
-      /*
-       * Some errors (e.g. HSQL constraint violations) seem to cause the
-       * rollback to fail. Extra error handling is needed here, too, to
-       * ensure that the failing data is captured.
-       */
```

Comment:
Was this hsql-specific handling removed because the transaction handler handles it for us, or because we've switched to using postgres as our primary testing db? (Asking since hsql is still supported so wondering if this still might need to exist)

Reply:
The `TransactionManager` takes care of handling errors during rollback so this code is obsolete.